### PR TITLE
Add `presser` to audits

### PR DESCRIPTION
**Pull request by fu5ha**
Does what it says on the tin :)

### CODE REVIEW

**Review by repi — APPROVED** [review]
thank you!

File: audits.toml [hunk 1]
```diff
@@ -102,6 +102,17 @@ criteria = "safe-to-deploy"
 version = "0.8.0"
 notes = "Maintained by the Ark team at Embark. Note that this is a unsafe FFI layer on top of a complex C++ library so may be some unknown unsoundness, but we have been using it in games production for a couple of years"
 
+[[audits.puffin]]
```
Comment (lqd) [inline audits.toml:105]: was this meant to be:

```suggestion
[[audits.presser]]
```

with version 0.4.0 rather than 0.13.0 ?

Comment (repi) [reply audits.toml:105]: oh wow, good catch. I totally missed that also when reading quickly.

Comment (fu5ha) [reply audits.toml:105]: 🤦‍♀️ oops, yes

**Pull request merged**